Dockerfile

```diff
@@ -7,7 +7,17 @@ COPY . /cerenetwork
 
 RUN apt-get update && \
 	  apt-get upgrade -y && \
-	  apt-get install -y cmake pkg-config libssl-dev git clang
+	  apt-get install -y cmake pkg-config libssl-dev git clang unzip
+
+# Installation script is taken from https://grpc.io/docs/protoc-installation/
+ENV PROTOC_VERSION=3.15.8
+RUN PB_REL="https://github.com/protocolbuffers/protobuf/releases" && \
```

Comment:
Please let me know where the dependency comes from.

Reply:
Some of the updated packages to `polkadot-v0.9.30` require `protoc`. An example of such package is `sc-network-bitswap`